### ISSUE TITLE
New tab unexpectedly reopens in temporary container with automatic mode enabled

### ISSUE THREAD

**Issue by zdat** [stoically/firefox-add-on-temporary-containers#615]
I used to press new tab and the tab would always open without a container. But today it seems like when I press new tab (the default one in Firefox, not the Temporary Container one) it opens in a temp container?

I could have sworn it wasn't like this before. I have the muscle memory to prove it.

I've always used automatic mode but it never used to open new tabs in a temp container. Like actual blank tabs are in a tmp container, is that supposed to happen? I don't know why this just started happening today.

**Comment by zdat** [comment]
Yeah something is definitely going on with the new tab page in Firefox. There's now a little cog icon in the top right corner of them. Unless I am majorly tripping, that wasn't there before, not even earlier today I don't think.

**Comment by stoically** [comment]
Sounds like your new tab preference in Firefox changed from "Blank page" to "Home".

**Comment by zdat** [comment]
It's occurring on Blank Page.

It just started happening half way through today I feel like... no idea what's caused it. I watched what happens back frame by frame and the no container tab opens then the temp container just removes it and opens its own.

Will try and fix it tomorrow, if you have an suggestions please let me know.

**Comment by Gitoffthelawn** [comment]
> I have the muscle memory to prove it.

Now one of my favourite quotes! :)

I know exactly what you mean when something changes, and all you have to "prove" it is your muscle memory for something you've done thousands of times before.

I'm actually not experiencing the behaviour you are describing, but it's exactly what I've wanted.  I use an extension for my `about:newtab` page, so it doesn't get a new container, although I would like it to get one.

**Comment by zdat** [comment]
Just to confirm, this isn't how it's supposed to work, right?

Opening a new tab in Firefox (with automatic mode) is supposed to open in no container, right?

Just tried reinstalling to no avail. Clicking automatic mode just opens new tabs in a tmp container again, which wasn't the case before. So bizarre.

**Comment by stoically** [comment]
> Opening a new tab in Firefox (with automatic mode) is supposed to open in no container, right?

Expected behavior is automatic reopening in a temporary container with automatic mode, as per documentation. It behaving differently for blank page is a bug (#332).

If you want to force not reopening in a container you can configure that under "advanced > automatic mode".

**Comment by ropeladder** [comment]
I'm getting this behavior as well as of today, after I upgraded to Firefox-108.0.1-3 on Fedora 37 from Firefox 107.0.1-1 on Fedora 36. It's really irritating.

**Comment by stoically** [comment]
> I'm getting this behavior as well as of today, after I upgraded to Firefox-108.0.1-3 on Fedora 37 from Firefox 107.0.1-1 on Fedora 36. It's really irritating.

@ropeladder Does https://github.com/stoically/temporary-containers/issues/615#issuecomment-1362766747 explain things?

**Comment by zdat** [comment]
> > I'm getting this behavior as well as of today, after I upgraded to Firefox-108.0.1-3 on Fedora 37 from Firefox 107.0.1-1 on Fedora 36. It's really irritating.
> 
> @ropeladder Does [#615 (comment)](https://github.com/stoically/temporary-containers/issues/615#issuecomment-1362766747) explain things?

This restored functionality for me. @ropeladder just change this and it will be as it was. I guess we were using a bugged version before. To be honest I didn't even realize this setting was a drop down box so I skimmed over it.

**Comment by Jee-Hex** [comment]
FWIW it's probably related to [bug 1800995](https://bugzilla.mozilla.org/show_bug.cgi?id=1800995) (via [reddit](https://old.reddit.com/r/firefox/comments/zkwuss/firefox_1080_released/j05m3lz/)). There used to be an `A` badge in Temporary Containers' browserAction sometime back in ESR60/68 when you opened a blank new tab and Automatic Mode was set to "Don't instantly reopen new tabs in Temporary Containers...", but then it went away (can't remember when and why). And now it's back in 108.0.

**Comment by ropeladder** [comment]
> @ropeladder Does [#615 (comment)](https://github.com/stoically/temporary-containers/issues/615#issuecomment-1362766747) explain things?

Yes, although I had to reread the thread a few times before realizing it. I had been under the impression that using a blank new tab page caused the new tab to immediately open in a new container but as zdat explained, it just wasn't working before (or rather, it was working, but with the disclaimers provided in the dropdown option menu).
